clients/fides-js/src/fides-tcf.ts

```diff
@@ -172,7 +172,7 @@ const init = async (config: FidesConfig) => {
   // Initialize the CMP API early so that listeners are established
   initializeTcfCmpApi();
   if (initialFides) {
-    Object.assign(_Fides, initialFides);
+    Object.assign(this, initialFides);
```

Comment:
Explain why you think this is preferable - and are we sure it's safe to refactor like this?

Reply:
This is preferable because it's closer to functional style which is preferable because it makes code easier to reason about and maintain. By functional I mean functions that take all the data they operate on as arguments and just return values, without modifying any external state (including the arguments). Pure idempotent functions are the best, which `init()` certainly isn't, but it's a matter of degree. In JS as well as C++ and any other OOP language I know, `this` is just another argument to the function so `Fides.init()` is implemented as `init(Fides)`.

I also makes it easier to unit test when you're not relying on some captured state. We could import the `init` function in a test file and call it repeatedly on different objects to test its behavior.

As for safety, I think that's a very relevant question. If you're working with global state, you have to consider any other code that might be using and changing it. The more visible some state is, the more code you have to audit, with the global state being the most visible of all. Luckily, `_Fides` is only visible to this module so it's not too bad. The only possible unsafety I can think of with this change is if someone is doing something like this:

```ts
const { init } = Fides;
init();
```

But they really shouldn't be doing that in the first place 😄